### ISSUE TITLE
Cannot get config info through config command

### ISSUE THREAD

**Issue by kotakotik22** [kotakotik22/chaoscraft#6]
Running something like ``/chaoscraft config general autoSync`` will give an error instead of giving the information about the config

**Comment by kotakotik22** [comment]
this does not make any sense

```java
@Override // show info
public int run(CommandContext<CommandSource> context) {
     System.out.println("hi :(");
     System.out.println(getInfoMessage(context, getConfigValue().get()).getString());
     context.getSource().sendFeedback(getInfoMessage(context, getConfigValue().get()), false);
     return 1;
}
```

it prints the ``hi :(``, but it doesnt print the info
![image](https://user-images.githubusercontent.com/61428759/107119041-47d4ce80-6885-11eb-9f41-3f76e62cdc9e.png)

**Comment by kotakotik22** [comment]
im still confused, but a little less 
```java
public ITextComponent getInfoMessage(CommandContext<CommandSource> context, Object val) {
            return new StringTextComponent("hello world");
//            return new StringTextComponent("value of " + getName() + " is " + val + "\ndefault value: " + getDefault()
//                    + "\ndescription: " + getDescription());
        }
```
it does say "hello world" but when i change it to the actual information string, it doesnt send it, what the hell??
![image](https://user-images.githubusercontent.com/61428759/107119153-f711a580-6885-11eb-93d9-f83522bdb146.png)

**Comment by kotakotik22** [comment]
it was a casting problem i guess idfk

**Comment by kotakotik22** [comment]
fixed in 1.2
